### PR TITLE
Codesec secrets scan

### PR DESCRIPTION
**Pull request by anaghapamidi**
Removed the list of supported file types/extensions from secrets-scanning.adoc. 

@tsmithv11 - This change is a part of [this ticket](https://redlock.atlassian.net/browse/RLP-77343), can you please helps us validate if we scan all files matched to specific signatures?

### CODE REVIEW

**Comment by tsmithv11** [comment]
All of these changes look good. I would separate out the changes for secrets because we're waiting on approval. The rest can be added immediately.

**Comment by anaghapamidi** [comment]
> All of these changes look good. I would separate out the changes for secrets because we're waiting on approval. The rest can be added immediately.

@tsmithv11 - Should we park the edit for removal of the "supported extension list as well types/extensions in secrets-scanning.adoc." as well, until further notice? The other changes are updated.

File: code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc [hunk 1]
```diff
@@ -146,7 +146,7 @@ image::sca-10.png[width=600]
 
 ==== Supply Chain
 
-As a remediation for sub-dependent packages, you can view and analyze the dependency tree on *Code Security > Supply Chain*. If the packages have direct dependencies irrespective of their placement in the dependency tree, Prisma Cloud offers solutions to these vulnerabilities. Here you can also choose to remediate the vulnerability by submitting a single PR (Pull Request) for all packages with vulnerabilities on the graph.
+As a remediation for sub-dependent packages, you can view and analyze the dependency tree on *Code Security > xref:../supply-chain-security.adoc[Supply Chain]*. If the packages have direct dependencies irrespective of their placement in the dependency tree, Prisma Cloud offers solutions to these vulnerabilities. Here you can also choose to remediate the vulnerability by submitting a single PR (Pull Request) for all packages with vulnerabilities on the graph.
```
Comment (Pubs-MV) [inline code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc:155]: @anaghapamidi did adding the xref in the UI path work?

Please confirm that you were able to run panconv and create a PDF without errors

Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/software-composition-analysis/software-composition-analysis.adoc:155]: @Pubs-MV yes the xref in the UI path works and the PDF is being generated smoothly.

**Review by try-panwiac — APPROVED** [review]

**Review by tsmithv11 — APPROVED** [review]

**Comment by Pubs-MV** [comment]
@anaghapamidi please resolve the conflicts, so we can merge. Also add the tested label to confirm that the content builds.

**Comment by anaghapamidi** [comment]
Hi @Pubs-MV I have resolved the conflicts. Request your review and approval. 

cc: @tsmithv11

**Pull request merged**